apps/backend/src/modules/groups/group.controller.ts

```diff
@@ -61,6 +61,15 @@ class GroupController extends BaseController {
 			},
 		});
 
+		this.addRoute({
+			handler: (options) =>
+				this.deleteGroup(
+					options as APIHandlerOptions<{ params: { id: number } }>,
+				),
+			method: "POST",
```

Comment:
The method should be "DELETE"

Reply:
thanks, I've already fixed it, just haven't pushed yet (electricity)